interfaces/cython/cantera/examples/onedim/diffusion_flame_extinction.py

```diff
@@ -114,6 +114,7 @@
     strain_factor = alpha[-1] / alpha[n_last_burning]
     # Create an initial guess based on the previous solution
     # Update grid
+    # NB: Grid scaling changes the diffusion flame width
```

Comment:
As a minor suggestion to improve readability, we're trying to avoid use of some more obscure abbreviations, especially these Latin ones, so I would just write "note" here.

Reply:
I guess 'NB' is a habit I picked up in British boarding school that I use without thinking about much :-)   I will note well that it is now arcane and obscure!